tests/next_tests/integration_tests/feature_tests/ffront_tests/test_arg_call_interface.py

```diff
@@ -271,3 +271,84 @@ def testee_scan(
         @program
         def testee(qc: cases.IKFloatField, param_1: int32, param_2: float, scalar: float):
             testee_scan(qc, param_1, param_2, scalar, out=(qc, param_1, param_2))
+
+
+def test_bound_args_wrong_kwargs(cartesian_case):
```

Comment:
It should be fairly easy to parameterize this test such that you only need one test case instead of 4. As a reference: `test_accept_args` has a similar pattern, though slightly more complicated than what is required here.